src/pointers.cpp

```diff
@@ -1644,7 +1644,7 @@ namespace big
         // Nullsub
         {
             "NS",
-            "90 C3",
+            "C3",
```

Comment:
This could have the same problem we're having now, where it's finding an encrypted chunk of memory that's later decrypted and no longer contains the RETN instruction. Also, wasn't there a significant reason we were using NOP RETN?